cmd/clusterawsadm/cloudformation/bootstrap/cluster_api_node.go

```diff
@@ -48,6 +48,19 @@ func (t Template) sessionManagerPolicy() iamv1.StatementEntry {
 	}
 }
 
+func (t Template) nodeManagedPolicies() []string {
+	polices := t.Spec.Nodes.ExtraPolicyAttachments
+
+	if t.Spec.Nodes.EKS.Enable {
+		polices = append(polices,
+			"arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy",
+			"arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy",
+			"arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly")
```

Comment:
`AmazonEC2ContainerRegistryReadOnly` would be a pretty useful toggle on the nodes level as it's not EKS specific.

If `AmazonEKS_CNI_Policy` is basically equivalent to that required by the VPC CNI, maybe also at the node level (and then use defaulter-gen to enable it when EKS is on for nodes).

Reply:
Good thinking, i'll make those changes.